changelog/preview-flags.dd

```diff
@@ -0,0 +1,84 @@
+`-preview` and `-revert` flags have been introduced
+
+For a long time new features have been added with little oversight and thus
+everytime a new, individual CLI flag got added.
+Over the time this has grown to:
+
+$(UL
+    $(LI `-dip25`)
+    $(LI `-dip1000`)
+    $(LI `-dip1000`)
+    $(LI `-dip1008`)
+)
+
+Moreover, in parallel `-transition` has been introduced and the following DMD
+example is only a subset of the transitional features that DMD supports at the moment:
+
+$(CONSOLE
+dmd -transition=3449 -transition=10378 -transition=14246 -transition=14888 -transition=16997
+)
+
+While these transitions also have a name alias, it's still not clear for the user
+whether the transition is (a) an experimental feature or an upcoming breaking change,
+(b) a warning or help on an upcoming change or (c) revert of a change for users
+who don't want to deal with the breaking change for now.
+
+With this release, DMD gained `-preview` which can be used to test upcoming features or potentially breaking changes.
+For example, with this release the list of upcoming features is:
+
+$(UL
+    $(LI `-preview=dip25`: Implements $(LINK2 https://github.com/dlang/DIPs/blob/master/DIPs/archive/DIP25.md, DIP25) (Sealed references))
+    $(LI `-preview=dip1000`: Implements $(LINK2 https://github.com/dlang/DIPs/blob/master/DIPs/DIP1000.md, DIP1000) (Scoped Pointers))
+    $(LI `-preview=dip1008`: Implements $(LINK2 https://github.com/dlang/DIPs/blob/master/DIPs/DIP1008.md, DIP1008) (`@nogc` Throwables))
+    $(LI `-preview=markdown`: Enables Markdown replacements in Ddoc)
+    $(LI `-preview=fixAliasThis`: Enables a potentially breaking fix for `alias this` with which DMD will first check the this scope before searching in upper scopes)
+    $(LI `-preview=intpromote`: Enables integral promotion for the unary `+`, `-` and `~` operators)
+    $(LI `-preview=dtorfields`: Enables a potentially breaking fix which enables to destruct fields of partially constructed objects)
+)
+
+It is very easy for the D development team to add new `-preview` features and, thus, this is often used to experiment with new features like they are proposed in DIPs.
```

Comment:
Adding new features can be disruptive to existing code bases. By initially putting them behind
a `-preview` switch users will have ample opportunity to adapt to them at their own pace.